examples/classic/sequential/BinSearch/BinSearch.qnt

```diff
@@ -2,7 +2,7 @@
 /**
  * A tutorial version of binary search. The original TLA+ specification:
  *
- * https://github.com/informalsystems/apalache/blob/unstable/test/tla/bin-search/BinSearch10.tla
+ * https://github.com/informalsystems/apalache/blob/unstable/test/tla/bin-search/BinSearch.tla
```

Comment:
The link is broken,, probably also in other files?

Reply:
Also note that `unstable` should probably be `main`?

Reply:
ah, all true :)